atd-etl/cris_import/cris_import.py

```diff
@@ -91,10 +106,12 @@ def main():
             pgloader_command_files = pgloader_csvs_into_database(schema_name)
             trimmed_token = remove_trailing_carriage_returns(pgloader_command_files)
             typed_token = align_db_typing(trimmed_token)
+            #typed_token = mess_with_incoming_records_to_ensure_updates(typed_token) # for testing PR 1316
```

Comment:
This is to help test this PR. I'll remove this one line before merging.

This function will add a random string to a field in each of the tables we're importing. This ensures that every line that comes in from CRIS is an update.

Reply:
This was really helpful for testing this out. 🙏 What do you think about moving the function called here to another module? I can see it coming in handy again the future, but I initially thought it was part of the fix so I'm not sure if it should live with the rest of the import code.

Reply:
I was going to ask if you were removing this line, if you would also remove the function, but I like Mike's idea about moving the function to another module for testing purposes.

Reply:
Thanks y'all for the great suggestions on this. I have pulled the testing function down into `lib/testing.py` with a little blurb in there about usage. This also required pulling a class I had tucked in the bottom of the main script out into a file, so that's what i'm up with with lib/sshkeytempdir.py.